### PR TITLE
User mover user model splitting - Part II

### PR DESCRIPTION
**Pull request by Kartones**
Closes https://github.com/CartoDB/cartodb/issues/5922

Relates to (and continues)  https://github.com/CartoDB/cartodb/pull/5898

### CODE REVIEW

**Comment by Kartones** [comment]
@juanignaciosl CR please.

Part 2 of the methods moving, this time no UserMover changes. Done quickly so I might have left some method interesting to move, but calls to `.db_service.xxx` are pretty low from main `User` model, and while indirections, names etc. might be improved... User has in total lost around 1200 LOC and now has some shared calls, with user mover, etc.


PS: I'm stabilizing tests, so initially either no more changes or just related to spec stabiliation.

**Comment by Kartones** [comment]
retest this please

**Comment by Kartones** [comment]
Ignoring check status, Branch is finally ok: http://ci.cartodb.com/job/CartoDB-PR-testing/1533/console

Proceeding to test at staging

File: app/models/data_import.rb [hunk 1]
```diff
@@ -624,8 +624,9 @@ def new_importer
       quota_checker = CartoDB::QuotaChecker.new(current_user)
       database      = current_user.in_database
       destination_schema = current_user.database_schema
```
Comment (juanignaciosl) [inline app/models/data_import.rb]: I hadn't seen this. We shouldn't be using `current_user` in a model, should we? Not something you should change, but if you agree I'll open an issue.

Comment (Kartones) [inline app/models/data_import.rb]: +1 is really old code but agreed that we shouldn't, at least not in this one that is so critical

File: lib/tasks/create_importer_schema.rake [hunk 2]
```diff
@@ -7,8 +7,8 @@ namespace :cartodb do
       ::User.all.each_with_index do |user, index|
         begin
           puts "Creating importer schema for #{user.username}"
-          user.create_importer_schema
-          user.db_service.set_user_privileges_in_importer_schema
+          user.db_service.create_importer_schema
+          user.db_service.db_service.set_user_privileges_in_importer_schema
```
Comment (juanignaciosl) [inline lib/tasks/create_importer_schema.rake]: `db_service.db_service.` looks duplicated.

**Comment by juanignaciosl** [comment]
:+1: after duplication probable fix.

**Pull request merged**